Index: amp-utils-1.3.7p/VisualStudioNET/Makefile
===================================================================
--- amp-utils-1.3.7p.orig/VisualStudioNET/Makefile	2011-09-29 18:12:43.000000000 +0200
+++ amp-utils-1.3.7p/VisualStudioNET/Makefile	2011-09-29 18:12:50.000000000 +0200
@@ -25,10 +25,10 @@
 
 all compile: ${TOOLS}
 library:
-#	su root -c "install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN}"
+#	$(call suInstall, install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN})
 scripts manuals:
 install: compile library
-#	su root -c "install -m 0555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN}"
+#	$(call suInstall, install -m 0555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN})
 uninstall:
 #	su root "cd -c "${BIN}; rm -f ${TOOLS}"
 #	su root "cd -c "${MAN}; rm -f ${PAGES}"
Index: amp-utils-1.3.7p/applets/Makefile
===================================================================
--- amp-utils-1.3.7p.orig/applets/Makefile	2011-09-29 18:12:43.000000000 +0200
+++ amp-utils-1.3.7p/applets/Makefile	2011-09-29 18:12:50.000000000 +0200
@@ -31,11 +31,11 @@
 
 all compile:
 manuals:
-	su root -c "install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN}"
+	$(call suInstall, install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN})
 scripts library install:
 uninstall:
-#	su root -c "cd ${BIN}; rm -f int6k ${TOOLS}"
-	su root -c "cd ${MAN}; rm -f int6k ${PAGES}"
+#	$(call suInstall, cd ${BIN}; rm -f int6k ${TOOLS})
+	$(call suInstall, cd ${MAN}; rm -f int6k ${PAGES})
 fresh: clean compile
 confirm:
 	${SH} applets.sh
Index: amp-utils-1.3.7p/classes/Makefile
===================================================================
--- amp-utils-1.3.7p.orig/classes/Makefile	2011-09-29 18:12:43.000000000 +0200
+++ amp-utils-1.3.7p/classes/Makefile	2011-09-29 18:12:50.000000000 +0200
@@ -34,14 +34,14 @@
 	if [ ! -z ${STRIP} ]; then ${STRIP} ${TOOLS}; fi
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 library:
-	su root -c "install -m 0755 -o 0 -g 0 -d ${BIN}"
+	$(call suInstall, install -m 0755 -o 0 -g 0 -d ${BIN})
 scripts:
 manuals:
-	su root -c "install -m 0644 -o 0 -g 0 ${PAGES} ${MAN}"
+	$(call suInstall, install -m 0644 -o 0 -g 0 ${PAGES} ${MAN})
 install: compile
-	su root -c "install -m 0555 -o 0 -g 0 ${TOOLS} ${BIN}"
+	$(call suInstall, install -m 0555 -o 0 -g 0 ${TOOLS} ${BIN})
 uninstall:
-	su root -c "cd ${BIN}; rm -f ${TOOLS}"
+	$(call suInstall, cd ${BIN}; rm -f ${TOOLS})
 fresh: clean compile
 confirm:
 	${SH} classes.sh
Index: amp-utils-1.3.7p/crypt/Makefile
===================================================================
--- amp-utils-1.3.7p.orig/crypt/Makefile	2011-09-29 18:12:43.000000000 +0200
+++ amp-utils-1.3.7p/crypt/Makefile	2011-09-29 18:12:50.000000000 +0200
@@ -34,15 +34,15 @@
 	if [ ! -z ${STRIP} ]; then ${STRIP} ${TOOLS}; fi
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 library:
-	su root -c "install -m 0755 -o 0 -g 0 -d ${BIN}"
+	$(call suInstall, install -m 0755 -o 0 -g 0 -d ${BIN})
 scripts:
 manuals:
-	su root -c "install -m 0755 -o 0 -g 0 ${PAGES} ${MAN}"
+	$(call suInstall, install -m 0755 -o 0 -g 0 ${PAGES} ${MAN})
 install: compile
-	su root -c "install -m 0555 -o 0 -g 0 ${TOOLS} ${BIN}"
+	$(call suInstall, install -m 0555 -o 0 -g 0 ${TOOLS} ${BIN})
 uninstall:
-	su root -c "cd ${BIN}; rm -f ${TOOLS}"
-	su root -c "cd ${MAN}; rm -f ${PAGES}"
+	$(call suInstall, cd ${BIN}; rm -f ${TOOLS})
+	$(call suInstall, cd ${MAN}; rm -f ${PAGES})
 fresh: clean compile
 confirm:
 	${SH} crypt.sh
Index: amp-utils-1.3.7p/docbook/Makefile
===================================================================
--- amp-utils-1.3.7p.orig/docbook/Makefile	2011-09-29 18:12:43.000000000 +0200
+++ amp-utils-1.3.7p/docbook/Makefile	2011-09-29 18:12:50.000000000 +0200
@@ -35,14 +35,14 @@
 	manuals
 scripts:
 library:
-	su root -c "install -m 0755 -o ${OWNER} -g ${GROUP} -d ${LIBRARY}"
+	$(call suInstall, install -m 0755 -o ${OWNER} -g ${GROUP} -d ${LIBRARY})
 install: compile library manuals
-	su root -c "if [ -d ${LIBRARY} ]; then install -m 0664 -o ${OWNER} -g ${GROUP} *.html *.css *.png ${LIBRARY}; fi"
-	su root -c "if [ -d ${LIBRARY} ]; then install -m 0664 -o ${OWNER} -g ${GROUP} piboffset.xsd piboffset.xml ${LIBRARY}; fi"
+	$(call suInstall, if [ -d ${LIBRARY} ]; then install -m 0664 -o ${OWNER} -g ${GROUP} *.html *.css *.png ${LIBRARY}; fi)
+	$(call suInstall, if [ -d ${LIBRARY} ]; then install -m 0664 -o ${OWNER} -g ${GROUP} piboffset.xsd piboffset.xml ${LIBRARY}; fi)
 uninstall:
-	su root -c "rm -f ${LIBRARY}/*.c.html ${LIBRARY}/*.h.html ${LIBRARY}/*.7.html"
+	$(call suInstall, rm -f ${LIBRARY}/*.c.html ${LIBRARY}/*.h.html ${LIBRARY}/*.7.html)
 setup:
-	su root -c "install ${STYLES} ${DOCBOOK}"
+	$(call suInstall, install ${STYLES} ${DOCBOOK})
 clean:
 	chmod 0644 ${PAGES} ${STYLES}
 	rm -f ${TRASH} ${BOOKS}
Index: amp-utils-1.3.7p/ether/Makefile
===================================================================
--- amp-utils-1.3.7p.orig/ether/Makefile	2011-09-29 18:12:43.000000000 +0200
+++ amp-utils-1.3.7p/ether/Makefile	2011-09-29 18:12:50.000000000 +0200
@@ -35,14 +35,14 @@
 	if [ ! -z ${STRIP} ]; then ${STRIP} ${TOOLS}; fi
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 library:
-	su root -c "install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN}"
+	$(call suInstall, install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN})
 scripts:
 manuals:
-	su root -c "install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN}"
+	$(call suInstall, install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN})
 install: compile library
-	su root -c "install -m 6555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN}"
+	$(call suInstall, install -m 6555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN})
 uninstall:
-	su root -c "cd ${BIN}; rm -f ${TOOLS}"
+	$(call suInstall, cd ${BIN}; rm -f ${TOOLS})
 fresh: clean compile
 confirm:
 	${SH} ether.sh
Index: amp-utils-1.3.7p/int6k2/Makefile
===================================================================
--- amp-utils-1.3.7p.orig/int6k2/Makefile	2011-09-29 18:12:43.000000000 +0200
+++ amp-utils-1.3.7p/int6k2/Makefile	2011-09-29 18:12:50.000000000 +0200
@@ -34,14 +34,14 @@
 	if [ ! -z ${STRIP} ]; then ${STRIP} ${TOOLS}; fi
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 library:
-	su root -c "install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN}"
+	$(call suInstall, install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN})
 scripts:
 manuals:
-	su root -c "install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN}"
+	$(call suInstall, install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN})
 install: compile library
-	su root -c "install -m 4555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN}"
+	$(call suInstall, install -m 4555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN})
 uninstall:
-	su root -c "cd ${BIN}; rm -f int6k ${TOOLS}"
+	$(call suInstall, cd ${BIN}; rm -f int6k ${TOOLS})
 clean:
 	rm -f ${TRASH} ${TOOLS}
 confirm:
Index: amp-utils-1.3.7p/make.def
===================================================================
--- amp-utils-1.3.7p.orig/make.def	2011-09-29 18:07:41.000000000 +0200
+++ amp-utils-1.3.7p/make.def	2011-09-29 18:15:58.000000000 +0200
@@ -81,3 +81,6 @@
 OWNER=0
 GROUP=0
 
+define suInstall
+  $(call suInstall, $(1))
+endef
Index: amp-utils-1.3.7p/mdio/Makefile
===================================================================
--- amp-utils-1.3.7p.orig/mdio/Makefile	2011-09-29 18:12:43.000000000 +0200
+++ amp-utils-1.3.7p/mdio/Makefile	2011-09-29 18:12:50.000000000 +0200
@@ -34,15 +34,15 @@
 	if [ ! -z ${STRIP} ]; then ${STRIP} ${TOOLS}; fi
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 library:
-	su root -c "install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN}"
+	$(call suInstall, install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN})
 scripts:
 manuals:
-	su root -c "install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN}"
+	$(call suInstall, install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN})
 install: compile library
-	su root -c "install -m 4555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN}"
+	$(call suInstall, install -m 4555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN})
 uninstall:
-	su root -c "cd ${BIN}; rm -f int6k ${TOOLS}"
-	su root -c "cd ${MAN}; rm -f int6k ${PAGES}"
+	$(call suInstall, cd ${BIN}; rm -f int6k ${TOOLS})
+	$(call suInstall, cd ${MAN}; rm -f int6k ${PAGES})
 clean:
 	chmod 0644 *.c *.h *.7 *.txt *.mak
 	chmod 0755 Makefile *.sh
Index: amp-utils-1.3.7p/mme/Makefile
===================================================================
--- amp-utils-1.3.7p.orig/mme/Makefile	2011-09-29 18:12:43.000000000 +0200
+++ amp-utils-1.3.7p/mme/Makefile	2011-09-29 18:12:50.000000000 +0200
@@ -34,14 +34,14 @@
 	if [ ! -z ${STRIP} ]; then ${STRIP} ${TOOLS}; fi
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 library:
-	su root -c "install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN}"
+	$(call suInstall, install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN})
 scripts:
 manuals:
-	su root -c "install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN}"
+	$(call suInstall, install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN})
 install: compile library
-	su root -c "install -m 06555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN}"
+	$(call suInstall, install -m 06555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN})
 uninstall:
-	su root -c "cd ${BIN}; rm -f ${TOOLS}"
+	$(call suInstall, cd ${BIN}; rm -f ${TOOLS})
 fresh: clean compile
 confirm:
 	${SH} mme.sh
Index: amp-utils-1.3.7p/nda/Makefile
===================================================================
--- amp-utils-1.3.7p.orig/nda/Makefile	2011-09-29 18:12:43.000000000 +0200
+++ amp-utils-1.3.7p/nda/Makefile	2011-09-29 18:12:50.000000000 +0200
@@ -34,15 +34,15 @@
 	if [ ! -z ${STRIP} ]; then ${STRIP} ${TOOLS}; fi
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 library:
-	su root -c "install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN}"
+	$(call suInstall, install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN})
 scripts:
 manuals:
-	su root -c "install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN}"
+	$(call suInstall, install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN})
 install: compile library
-	su root -c "install -m 4555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN}"
+	$(call suInstall, install -m 4555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN})
 uninstall:
-	su root -c "cd ${BIN}; rm -f plc ${TOOLS}"
-	su root -c "cd ${MAN}; rm -f plc ${PAGES}"
+	$(call suInstall, cd ${BIN}; rm -f plc ${TOOLS})
+	$(call suInstall, cd ${MAN}; rm -f plc ${PAGES})
 fresh: clean compile
 confirm:
 	${SH} nda.sh
Index: amp-utils-1.3.7p/nodes/Makefile
===================================================================
--- amp-utils-1.3.7p.orig/nodes/Makefile	2011-09-29 18:12:43.000000000 +0200
+++ amp-utils-1.3.7p/nodes/Makefile	2011-09-29 18:12:50.000000000 +0200
@@ -27,12 +27,12 @@
 	if [ ! -z ${STRIP} ]; then ${STRIP} ${TOOLS}; fi
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 library:
-#	su root -c "install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN}"
+#	$(call suInstall, install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN})
 scripts manuals:
 install: compile library
-#	su root -c "install -m 0555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN}"
+#	$(call suInstall, install -m 0555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN})
 uninstall:
-#	su root -c "cd ${BIN}; rm -f ${TOOLS}"
+#	$(call suInstall, cd ${BIN}; rm -f ${TOOLS})
 fresh: clean compile
 confirm:
 	${SH} nodes.sh
Index: amp-utils-1.3.7p/nvm/Makefile
===================================================================
--- amp-utils-1.3.7p.orig/nvm/Makefile	2011-09-29 18:12:43.000000000 +0200
+++ amp-utils-1.3.7p/nvm/Makefile	2011-09-29 18:12:50.000000000 +0200
@@ -35,13 +35,13 @@
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 scripts:
 manuals:
-	su root -c "install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN}"
+	$(call suInstall, install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN})
 library:
-	su root -c "install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN}"
+	$(call suInstall, install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN})
 install: compile library
-	su root -c "install -m 0555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN}"
+	$(call suInstall, install -m 0555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN})
 uninstall:
-	su root -c "cd ${BIN}; rm -f ${TOOLS}"
+	$(call suInstall, cd ${BIN}; rm -f ${TOOLS})
 fresh: clean compile
 confirm:
 	${SH} nvm.sh
Index: amp-utils-1.3.7p/pib/Makefile
===================================================================
--- amp-utils-1.3.7p.orig/pib/Makefile	2011-09-29 18:12:43.000000000 +0200
+++ amp-utils-1.3.7p/pib/Makefile	2011-09-29 18:12:50.000000000 +0200
@@ -35,13 +35,13 @@
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 scripts:
 manuals:
-	su root -c "install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN}"
+	$(call suInstall, install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN})
 library:
-	su root -c "install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN}"
+	$(call suInstall, install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN})
 install: compile library
-	su root -c "install -m 0555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN}"
+	$(call suInstall, install -m 0555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN})
 uninstall:
-	su root -c "cd ${BIN}; rm -f ${TOOLS}"
+	$(call suInstall, cd ${BIN}; rm -f ${TOOLS})
 fresh: clean compile
 confirm:
 	${SH} pib.sh
Index: amp-utils-1.3.7p/plc/Makefile
===================================================================
--- amp-utils-1.3.7p.orig/plc/Makefile	2011-09-29 18:12:43.000000000 +0200
+++ amp-utils-1.3.7p/plc/Makefile	2011-09-29 18:12:50.000000000 +0200
@@ -38,15 +38,15 @@
 	if [ ! -z ${STRIP} ]; then ${STRIP} ${TOOLS}; fi
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 library:
-	su root -c "install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN}"
+	$(call suInstall, install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN})
 scripts:
 manuals:
-	su root -c "install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN}"
+	$(call suInstall, install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN})
 install: compile library
-	su root -c "install -m 4555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN}"
+	$(call suInstall, install -m 4555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN})
 uninstall:
-	su root -c "cd ${BIN}; rm -f int6k ${TOOLS}"
-	su root -c "cd ${MAN}; rm -f int6k ${PAGES}"
+	$(call suInstall, cd ${BIN}; rm -f int6k ${TOOLS})
+	$(call suInstall, cd ${MAN}; rm -f int6k ${PAGES})
 fresh: clean compile
 confirm:
 	${SH} plc.sh
Index: amp-utils-1.3.7p/programs/Makefile
===================================================================
--- amp-utils-1.3.7p.orig/programs/Makefile	2011-09-29 18:12:43.000000000 +0200
+++ amp-utils-1.3.7p/programs/Makefile	2011-09-29 18:12:50.000000000 +0200
@@ -35,14 +35,14 @@
 	if [ ! -z ${STRIP} ]; then ${STRIP} ${TOOLS}; fi
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 library:
-	su root -c "install -m 0755 -o 0 -g 0 -d ${BIN}"
+	$(call suInstall, install -m 0755 -o 0 -g 0 -d ${BIN})
 scripts:
 manuals:
-	su root -c "install -m 0644 -o 0 -g 0 ${PAGES} ${MAN}"
+	$(call suInstall, install -m 0644 -o 0 -g 0 ${PAGES} ${MAN})
 install: compile
-	su root -c "install -m 4555 -o 0 -g 0 ${TOOLS} ${BIN}"
+	$(call suInstall, install -m 4555 -o 0 -g 0 ${TOOLS} ${BIN})
 uninstall:
-	su root -c "root -c cd ${BIN}; rm -f ${TOOLS}"
+	$(call suInstall, root -c cd ${BIN}; rm -f ${TOOLS})
 fresh: clean compile
 confirm:
 	${SH} programs.sh
Index: amp-utils-1.3.7p/ram/Makefile
===================================================================
--- amp-utils-1.3.7p.orig/ram/Makefile	2011-09-29 18:12:43.000000000 +0200
+++ amp-utils-1.3.7p/ram/Makefile	2011-09-29 18:12:50.000000000 +0200
@@ -34,15 +34,15 @@
 	if [ ! -z ${STRIP} ]; then ${STRIP} ${TOOLS}; fi
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 library:
-	su root -c "install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN}"
+	$(call suInstall, install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN})
 scripts:
 manuals:
-	su root -c "install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN}"
+	$(call suInstall, install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN})
 install: compile library
-	su root -c "install -m 0555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN}"
+	$(call suInstall, install -m 0555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN})
 uninstall:
-	su root -c "cd ${BIN}; rm -f ${TOOLS}"
-	su root -c "cd ${MAN}; rm -f ${PAGES}"
+	$(call suInstall, cd ${BIN}; rm -f ${TOOLS})
+	$(call suInstall, cd ${MAN}; rm -f ${PAGES})
 fresh: clean compile
 confirm:
 	${SH} ram.sh
Index: amp-utils-1.3.7p/scripts/Makefile
===================================================================
--- amp-utils-1.3.7p.orig/scripts/Makefile	2011-09-29 18:12:43.000000000 +0200
+++ amp-utils-1.3.7p/scripts/Makefile	2011-09-29 18:12:50.000000000 +0200
@@ -31,7 +31,7 @@
 install:
 scripts:
 manuals:
-	su root -c "install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN}"
+	$(call suInstall, install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN})
 library:
 uninstall:
 clean:
Index: amp-utils-1.3.7p/serial/Makefile
===================================================================
--- amp-utils-1.3.7p.orig/serial/Makefile	2011-09-29 18:12:43.000000000 +0200
+++ amp-utils-1.3.7p/serial/Makefile	2011-09-29 18:12:50.000000000 +0200
@@ -34,15 +34,15 @@
 	if [ ! -z ${STRIP} ]; then ${STRIP} ${TOOLS}; fi
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 library:
-	su root -c "install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN}"
+	$(call suInstall, install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN})
 scripts:
 manuals:
-	su root -c "install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN}"
+	$(call suInstall, install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN})
 install: compile library
-	su root -c "install -m 0555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN}"
+	$(call suInstall, install -m 0555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN})
 uninstall:
-	su root -c "cd ${BIN}; rm -f int6k ${TOOLS}"
-	su root -c "cd ${MAN}; rm -f int6k ${PAGES}"
+	$(call suInstall, cd ${BIN}; rm -f int6k ${TOOLS})
+	$(call suInstall, cd ${MAN}; rm -f int6k ${PAGES})
 fresh: clean compile
 confirm:
 	${SH} serial.sh
Index: amp-utils-1.3.7p/snmp/Makefile
===================================================================
--- amp-utils-1.3.7p.orig/snmp/Makefile	2011-09-29 18:12:43.000000000 +0200
+++ amp-utils-1.3.7p/snmp/Makefile	2011-09-29 18:12:50.000000000 +0200
@@ -36,17 +36,17 @@
 	if [ ! -z ${STRIP} ]; then ${STRIP} ${TOOLS}; fi
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 library:
-#	su root -c "install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN}"
+#	$(call suInstall, install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN})
 scripts:
 manuals:
-#	su root -c "install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN}"
+#	$(call suInstall, install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN})
 install: compile
-	su root -c "install -m 4555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN}"
-	su root -c "install -m 755 -o ${OWNER} -g ${GROUP} -d ${MIB_DIR}"
-	su root -c "install -m 444 -o ${OWNER} -g ${GROUP} ${MIBS} ${MIB_DIR}"
+	$(call suInstall, install -m 4555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN})
+	$(call suInstall, install -m 755 -o ${OWNER} -g ${GROUP} -d ${MIB_DIR})
+	$(call suInstall, install -m 444 -o ${OWNER} -g ${GROUP} ${MIBS} ${MIB_DIR})
 uninstall:
-#	su root -c "cd ${BIN}; rm -f int6k ${TOOLS}"
-#	su root -c "cd ${MAN}; rm -f int6k ${PAGES}"
+#	$(call suInstall, cd ${BIN}; rm -f int6k ${TOOLS})
+#	$(call suInstall, cd ${MAN}; rm -f int6k ${PAGES})
 clean:
 	rm -f ${TRASH} ${TOOLS}
 test:
Index: amp-utils-1.3.7p/snmp2/Makefile
===================================================================
--- amp-utils-1.3.7p.orig/snmp2/Makefile	2011-09-29 18:12:43.000000000 +0200
+++ amp-utils-1.3.7p/snmp2/Makefile	2011-09-29 18:12:50.000000000 +0200
@@ -36,17 +36,17 @@
 	if [ ! -z ${STRIP} ]; then ${STRIP} ${TOOLS}; fi
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 #library:
-#	su root -c "install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN}"
+#	$(call suInstall, install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN})
 scripts:
 #manuals:
-#	su root -c "install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN}"
+#	$(call suInstall, install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN})
 install: compile
-	su root -c "install -m 4555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN}"
-	su root -c "install -m 755 -o ${OWNER} -g ${GROUP} -d ${MIB_DIR}"
-	su root -c "install -m 444 -o ${OWNER} -g ${GROUP} ${MIBS} ${MIB_DIR}"
+	$(call suInstall, install -m 4555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN})
+	$(call suInstall, install -m 755 -o ${OWNER} -g ${GROUP} -d ${MIB_DIR})
+	$(call suInstall, install -m 444 -o ${OWNER} -g ${GROUP} ${MIBS} ${MIB_DIR})
 #uninstall:
-#	su root -c "cd ${BIN}; rm -f int6k ${TOOLS}"
-#	su root -c "cd ${MAN}; rm -f int6k ${PAGES}"
+#	$(call suInstall, cd ${BIN}; rm -f int6k ${TOOLS})
+#	$(call suInstall, cd ${MAN}; rm -f int6k ${PAGES})
 clean:
 	rm -f ${TRASH} ${TOOLS}
 test:
Index: amp-utils-1.3.7p/tools/Makefile
===================================================================
--- amp-utils-1.3.7p.orig/tools/Makefile	2011-09-29 18:12:43.000000000 +0200
+++ amp-utils-1.3.7p/tools/Makefile	2011-09-29 18:12:50.000000000 +0200
@@ -34,16 +34,16 @@
 	if [ ! -z ${STRIP} ]; then ${STRIP} ${TOOLS}; fi
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 library:
-#	su root -c "install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN}"
+#	$(call suInstall, install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN})
 scripts:
 manuals:
-	su root -c "rm -f /usr/share/man/*/{ilt,tools}.*"
-	su root -c "install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN}"
+	$(call suInstall, rm -f /usr/share/man/*/{ilt,tools}.*)
+	$(call suInstall, install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN})
 install: compile library
-	su root -c "install -m 0555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN}"
+	$(call suInstall, install -m 0555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN})
 uninstall:
-	su root -c "cd ${BIN}; rm -f ${TOOLS}"
-	su root -c "cd ${MAN}; rm -f ${PAGES}"
+	$(call suInstall, cd ${BIN}; rm -f ${TOOLS})
+	$(call suInstall, cd ${MAN}; rm -f ${PAGES})
 fresh: clean compile
 confirm: 
 	${SH} tools.sh
Index: amp-utils-1.3.7p/toys/Makefile
===================================================================
--- amp-utils-1.3.7p.orig/toys/Makefile	2011-09-29 18:12:43.000000000 +0200
+++ amp-utils-1.3.7p/toys/Makefile	2011-09-29 18:12:50.000000000 +0200
@@ -34,15 +34,15 @@
 	if [ ! -z ${STRIP} ]; then ${STRIP} ${TOOLS}; fi
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 library:
-	su root -c "install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN}"
+	$(call suInstall, install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN})
 scripts:
 manuals:
-	su root -c "install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN}"
+	$(call suInstall, install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN})
 install: compile library
-	su root -c "install -m 4555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN}"
+	$(call suInstall, install -m 4555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN})
 uninstall:
-	su root -c "cd ${BIN}; rm -f int6k ${TOOLS}"
-	su root -c "cd ${MAN}; rm -f int6k ${PAGES}"
+	$(call suInstall, cd ${BIN}; rm -f int6k ${TOOLS})
+	$(call suInstall, cd ${MAN}; rm -f int6k ${PAGES})
 fresh: clean compile
 confirm:
 	${SH} toys.sh
Index: amp-utils-1.3.7p/tr69/Makefile
===================================================================
--- amp-utils-1.3.7p.orig/tr69/Makefile	2011-09-29 18:12:43.000000000 +0200
+++ amp-utils-1.3.7p/tr69/Makefile	2011-09-29 18:12:50.000000000 +0200
@@ -35,15 +35,15 @@
 	if [ ! -z ${STRIP} ]; then ${STRIP} ${TOOLS}; fi
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 library:
-	su root -c "install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN}"
+	$(call suInstall, install -m 0755 -o ${OWNER} -g ${GROUP} -d ${BIN})
 scripts manuals:
 manuals:
-#	su root -c "install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN}"
+#	$(call suInstall, install -m 0444 -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN})
 install: compile library
-	su root -c "install -m 0555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN}"
+	$(call suInstall, install -m 0555 -o ${OWNER} -g ${GROUP} -s ${TOOLS} ${BIN})
 uninstall:
-	su root -c "cd ${BIN}; rm -f ${TOOLS}"
-	su root -c "cd ${MAN}; rm -f ${PAGES}"
+	$(call suInstall, cd ${BIN}; rm -f ${TOOLS})
+	$(call suInstall, cd ${MAN}; rm -f ${PAGES})
 clean:
 	rm -f ${TOOLS} ${TRASH}
 test:
